test/JDBC/jdbc_schedule

```diff
@@ -11,6 +11,13 @@
 all
 
 # Temporarily ignore tests until fixed in PG15
```

Comment:
We should file a jira for these tests

Reply:
Created: https://jira.rds.a2z.com/browse/BABEL-3811